src/main/java/edu/cornell/kfs/concur/batch/ConcurRequestExtractPdpConstants.java

```diff
@@ -0,0 +1,23 @@
+package edu.cornell.kfs.concur.batch;
+
+public class ConcurRequestExtractPdpConstants {
+
+    public static class ValidationConstants {
+        public static final String CASH_ADVANCE_INDICATOR = "CASH ADVANCE";
+        public static final String EMPLOYEE_INDICATOR = "EMPLOYEE";
+        public static final String NON_EMPLOYEE_INDICATOR = "NON-EMPLOYEE";
```

Comment:
there are constants for employee and non-employee is ConcurConstants, can those be used?